examples/chip-tool/commands/common/CHIPCommand.h

```diff
@@ -62,6 +62,7 @@ class CHIPCommand : public Command
         AddArgument("paa-trust-store-path", &mPaaTrustStorePath);
         AddArgument("commissioner-name", &mCommissionerName);
         AddArgument("commissioner-nodeid", 0, UINT64_MAX, &mCommissionerNodeId);
+        AddArgument("commissioner-fabricid", 0, UINT64_MAX, &mCommissionerNodeId);
```

Comment:
```suggestion
        AddArgument("commissioner-fabricid", 0, UINT64_MAX, & mCommissionerFabricId);
```

Reply:
An additional question for my edification: what is the chip-tool cli interface to actually pass this parameter?

Reply:
`--commissioner-fabricid 124345` or `--commissioner-fabricd 0xabcdef1234567890`